src/main/java/org/broadinstitute/consent/http/resources/EmailNotifierResource.java

```diff
@@ -29,4 +30,17 @@ public Response sendReminderMessage(@PathParam("voteId") String voteId) {
       return createExceptionResponse(e);
     }
   }
+
+  @GET
+  @Path("/darExpirationNotices")
+  @RolesAllowed({SERVICE_ACCOUNT})
+  public Response sendDarExpirationNotices() {
+    try {
+      emailService.sendExpirationNotices();
```

Comment:
There are many ways to think about this, but I had been thinking that this operation was a `DataAccessRequestResource` operation. The operation is "check for expired or expiring requests". Depending on the check results, some emails may be sent (and some data in the DAR may be updated); but IMO this is primarily a DAR operation, not an email one.

Reply:
Funny, I think of this as "send emails to the people with DARs that are expired or expiring soon" and appreciate your perspective.  I could support a refactor ticket that moves both this method and the `sendReminderMessage` above it to their respective services to eliminate the need for this endpoint all together.